packages/veui/src/components/Table/_TableHead.js

```diff
@@ -1,10 +1,11 @@
 import Checkbox from '../Checkbox'
 import Sorter from './_Sorter'
-import Popover from '../Popover.vue'
+import Popover from '../Popover'
 import prefix from '../../mixins/prefix'
 import table from '../../mixins/table'
 import i18n from '../../mixins/i18n'
 import '../../common/uiTypes'
+import { isFocusable } from '@/utils/dom'
```

Comment:
```suggestion
import { isFocusable } from '../../utils/dom'
```

VEUI 会和业务代码一起 build，这个 `@` 的配置和业务代码不一样的。